.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+- '6'
+- '8'
+- '9'
+- '10'
+install: npm install
```

Comment:
Please use yarn for installing and running tests.

Reply:
Done